cardano-cli/test/cardano-cli-golden/files/input/drep.skey

```diff
@@ -0,0 +1,5 @@
+{
+    "type": "DRepSigningKey_ed25519",
+    "description": "Delegate Representative Signing Key",
+    "cborHex": "58201872d334e12a3a062dbdc4f2a22dc77f6e88ba8ecb39d0a2e385375f15ba0005"
+}
```

Comment:
It is proposed that test input files are put in a separate directory.  These can be re-used.  We don't need to have  separate keys of the same type for each test for example.  This means there shouldn't be many such files and we can throw them all in the same directory.